plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -222,6 +223,21 @@ public BigQueryConfig setServiceCacheTtl(Duration serviceCacheTtl)
         return this;
     }
 
+    @NotNull
+    @MinDuration("0m")
```

Comment:
Does a `MinDuration("0m")` also accept `0ms`? I think it would - important because docs say to set to `0ms` to disable cache.